pkg/common/configs/configvalidator.go

```diff
@@ -356,24 +356,20 @@ func checkPlacementRules(partition *PartitionConfig) error {
 	return nil
 }
 
-func checkQueueHierarchyForPlacement(path []string, create, hasDynamicPart bool, conf []QueueConfig, parentConf *QueueConfig) (result placementPathCheckResult, lastQueueName string) {
+func checkQueueHierarchyForPlacement(path []string, create, hasDynamicPart bool, conf []QueueConfig, parentConf *QueueConfig) (placementPathCheckResult, string) {
 	queueName := path[0]
-	lastQueueName = ""
```

Comment:
Hi @plsmaop,

Could we keep the variable lastQueueName alive in this function? My concern is for readability. When I look back at checkQueueHierarchyForPlacement(), it's quite difficult to fugure out that the second string is lastQueueName.

Reply:
sure

Reply:
added